lib/actions/artisan.js

```diff
@@ -184,19 +186,63 @@ module.exports = {
       return;
     }
 
-    // Log
-    Logs.show('server-started', {
-      type: 'Info',
-      message: null,
-      params: {}
+    // Grab the configurations
+    const config = JSON.parse(Utils.readFileSync('./site.json', 'utf-8'));
+    const portToSet = options.port || config.server.port || env.server.defaultPort;
+
+    // If the given port is different than whats in the config, update site.json with the new port
+    if (portToSet !== config.server.port) {
+      config.server.port = portToSet;
+      fsExtra.outputFileSync(`${env.templates.config}`, JSON.stringify(config, null, 2));
```

Comment:
Here, we have a method in '[actions/utils.js](https://github.com/kodeflex/wp-artisan/blob/46dfcb686eceaec1007c2de0eab2a5f76c12ae88/lib/actions/utils.js#L43)` as `Utils.writeFileSync()` :)

So, we can remove both, `mysql` and 'fs-extra` module from here and just call `Utils` and `Db` actions.